File: backend/server/models/css.js
function attributes(DataTypes) {
    return {
        selector: {
            type: DataTypes.STRING,
            allowNull: false
        },
        property: {
            type: DataTypes.STRING,
            allowNull: false
        },
        value: {
            type: DataTypes.STRING,
            allowNull: false
        },
        type: {
            type: DataTypes.STRING,
            allowNull: false
        },
        title: {
            type: DataTypes.STRING,
            allowNull: false
        }
    };
}

module.exports = function(sequelize, DataTypes) {
    return sequelize.define('Css', attributes(DataTypes));
}

File: backend/src/main/java/no/javazone/switcharoo/api/model/Css.java
package no.javazone.switcharoo.api.model;

public class Css {
    public final Long id;
    public final String selector;
    public final String property;
    public final String value;
    public final String type;
    public final String title;

    public Css(Long id, String selector, String property, String value, String type, String title) {
        this.id = id;
        this.selector = selector;
        this.property = property;
        this.value = value;
        this.type = type;
        this.title = title;
    }
}
